snakemake_wrapper_utils/snakemake.py

```
@@ -22,8 +22,25 @@ def get_mem(snakemake, out_unit="MiB"):
-    if arg in cmd.replace("=", " ").split(" "):
-        return True
+    """Check command for the presence of argument."""
+    return arg in cmd.replace("=", " ").split(" ")
 
-    return False
+
+def get_format(path):
+    """Get file format from extension, ignoring common compressions."""
+    if not path:
+        raise ValueError("Path cannot be empty")
+    exts = Path(path).suffixes
```

Ruff (0.8.0)

Undefined name `Path`

(F821)